src/ManTooltip/widget/ManTooltip.js

```diff
@@ -4,9 +4,21 @@ These buttons display a help message when clicked or hovered.
 
 Optionally, the buttons can be hidden by default, with a global switch (the Help Text Trigger) to show or hide them. 
 */
-dojo.provide("ManTooltip.widget.ManTooltip");
+define([
+    'dojo/_base/declare',
+    'mxui/widget/_WidgetBase',
+    'mxui/dom',
+    'dojo/_base/lang',

```

Comment:
When added code, stick our standard of using double quotes

Reply:
Done